running-recipes/getting-started.md

```diff
@@ -298,6 +298,8 @@ After doing that, your `pom.xml` file should look similar to this:
 
 To double-check that everything is working, run the command `./mvnw rewrite:run`. Your project should be upgraded to Spring Boot 2 and all of the test classes should be updated to JUnit 5. Your `pom.xml` file will also have had its Spring dependencies updated, the JUnit 4 dependency removed, and the JUnit 5 dependency added.
 
+__note__ here [bill of materials (BOM) dependency](https://docs.gradle.org/current/userguide/platforms.html#sub:bom\_import) is not mentioned for `dependencyManagement` as we can not use it to version control the OpenRewrite plugin dependencies, unlike with the Gradle setup below.
```

Comment:
Thanks for taking the time to write this!

Does this rephrase make sense to you? I wasn't sure if there was a benefit to linking to the Gradle BOM here or if calling out `dependencyManagement` would be useful. However, I could definitely be missing something. Open to editing this further!

Also, separately, GitBook provides a code snippet we can use to get stylized hints in the docs. I added that to be suggestion below.

```suggestion
{% hint style="info" %}
Maven does not currently support using a bill of materials (BOM) to specify plugin versions or dependencies. This means that you will have to specify the versions of each plugin by hand, unlike in the Gradle section below.
{% endhint %}
```

Reply:
That looks good to me, it captures the essence and those pondering this will know where the bom would be used.